src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -51,10 +54,11 @@ private static string GetConfigString(Config conf, string name)
             return value;
         }
 
-        public TestTransport(Address localAddress, AssociationRegistry registry, string schemeIdentifier = "test")
+        public TestTransport(Address localAddress, AssociationRegistry registry, long maximumPayloadBytes = 32000, string schemeIdentifier = "test")
```

Comment:
Why... yes it does. https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/transport/TestTransport.scala#L36

:+1: